include/bluetooth/services/otc.h

```diff
@@ -106,21 +123,25 @@ struct bt_otc_instance_t {
 	struct bt_otc_obj_metadata cur_object;
 };
```

Comment:
Should we call this structure `bt_otc` to be consistent with `bt_ots`?

Reply:
This is the metadata struct, not the instance struct.
The instance struct is renamed to bt_ots_client.